### PR TITLE
Feature/cluster

### PR DESCRIPTION
**Pull request by yduchesne**

### CODE REVIEW

File: modules/client/pom.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
 	<parent>
 		<groupId>org.sapia</groupId>
 		<artifactId>sapia_corus_parent</artifactId>
-		<version>develop-SNAPSHOT</version>
+		<version>5.0.5</version>
```
Comment (jcdesrochers) [inline modules/client/pom.xml]: Why version 5.0.5? Is this for the next release?

File: modules/server/src/main/java/org/sapia/corus/cluster/ClusterManagerImpl.java [hunk 2]
```diff
@@ -43,16 +46,17 @@
...
+  private static final int RECONNECTION_DELAY_OFFSET    = 2000;
 
-  private EventChannel                 channel;
-  private Set<CorusHost>               hostsInfos        = Collections.synchronizedSet(new HashSet<CorusHost>());
-  private Map<String, CorusHost>       hostsByNode       = Collections.synchronizedMap(new HashMap<String, CorusHost>());
-  private ServerSideClusterInterceptor interceptor;
-  private DeferredAsyncListener        deferredListeners = new DeferredAsyncListener();
-  private long                         startTime         = System.currentTimeMillis();
+  private EventChannel                  channel;
+  //private Map<ServerAddress, CorusHost> hostsByAddress    = Collections.synchronizedMap(new HashMap<ServerAddress, CorusHost>());
```
Comment (jcdesrochers) [inline modules/server/src/main/java/org/sapia/corus/cluster/ClusterManagerImpl.java]: Still need this comment?

File: samples/corus_sample_docker/pom.xml [hunk 3]
```diff
@@ -9,7 +9,11 @@
   <parent>
     <groupId>org.sapia</groupId>
     <artifactId>sapia_corus_samples_parent</artifactId>
-    <version>develop-SNAPSHOT</version>
+<<<<<<< HEAD
+    <version>5.0.4</version>
+=======
+    <version>5.0.5</version>
+>>>>>>> release/5.0.5
```
Comment (jcdesrochers) [inline samples/corus_sample_docker/pom.xml]: Merge conflict!

File: samples/corus_sample_grails_app/pom.xml [hunk 4]
```diff
@@ -6,7 +6,11 @@
 	<parent>
 		<groupId>org.sapia</groupId>
 		<artifactId>sapia_corus_samples_parent</artifactId>
-		<version>develop-SNAPSHOT</version>
+<<<<<<< HEAD
+		<version>5.0.4</version>
+=======
+		<version>5.0.5</version>
+>>>>>>> release/5.0.5
```
Comment (jcdesrochers) [inline samples/corus_sample_grails_app/pom.xml]: ditto

File: samples/corus_sample_jetty/pom.xml [hunk 5]
```diff
@@ -9,7 +9,11 @@
   <parent>
     <groupId>org.sapia</groupId>
     <artifactId>sapia_corus_samples_parent</artifactId>
-    <version>develop-SNAPSHOT</version>
+<<<<<<< HEAD
+    <version>5.0.4</version>
+=======
+    <version>5.0.5</version>
+>>>>>>> release/5.0.5
```
Comment (jcdesrochers) [inline samples/corus_sample_jetty/pom.xml]: ...

**Comment by yduchesne** [comment]
Outdated

**Pull request closed without merging**